app/src/main/java/io/github/incplusplus/potwhole/RegisterAccountPage.java

```diff
@@ -0,0 +1,81 @@
+package io.github.incplusplus.potwhole;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.util.Patterns;
+import android.widget.Button;
+import android.widget.EditText;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+import com.google.android.gms.tasks.OnFailureListener;
+import com.google.firebase.auth.FirebaseAuth;
+
+public class RegisterAccountPage extends AppCompatActivity {
+
+    private FirebaseAuth mAuth;
+
+    private EditText editTextEmail, editTextPassword;
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        mAuth = FirebaseAuth.getInstance();
+
+        setContentView(R.layout.activity_register_account);
+
+        editTextEmail = findViewById(R.id.editTextEmail);
+        editTextPassword = findViewById(R.id.editTextPassword);
+
+        Button RegisterAccountButton = findViewById(R.id.Register_Account_button);
+
+        RegisterAccountButton.setOnClickListener(v -> createUserAccount());
+    }
+
+    private void createUserAccount() {
+        String email = editTextEmail.getText().toString().trim();
+        String password = editTextPassword.getText().toString().trim();
+
+        if (email.isEmpty()) {
+            editTextEmail.setError("Please Enter an Email Address");
+            editTextEmail.requestFocus();
+            return;
+        }
+
+        if (!Patterns.EMAIL_ADDRESS.matcher(email).matches()) {
+            editTextEmail.setError("Please Enter a Valid Email Address");
+            editTextEmail.requestFocus();
+            return;
+        }
+
+        if (password.isEmpty()) {
+            editTextEmail.setError("Please Enter a Password");
+            editTextEmail.requestFocus();
+            return;
+        }
+
+        mAuth.createUserWithEmailAndPassword(email, password)
+                .addOnSuccessListener(
+                        authResult -> {
+                            System.out.println("User Account is Created in");
```

Comment:
Same story here. If you want to keep this around for debugging purposes, this should be switched out for a log statement.